task.go

```diff
@@ -28,15 +28,21 @@ var (
 	TaskSystemType = "system"
 )
 
+type contextKey string
```

Comment:
What does this type alias benefit? I don't see `contextKey` mentioned anywhere in the diff.

Reply:
staticcheck will complain if you don't use a special interface type. If you use a string directly, it complains about the possibility of conflicting with another package.